diff --git a/gtk-2.0/gtkrc b/gtk-2.0/gtkrc
index 37e837a..76ba623 100644
--- a/gtk-2.0/gtkrc
+++ b/gtk-2.0/gtkrc
@@ -1,32 +1,20 @@
 #modded Numix gtkrc
 
-gtk-color-scheme = "bg_color:#131971
-color0:#131971
-color1:#9E9BDE
-color2:#8F91D6
-color3:#AEB0EA
-color4:#5E6EDD
-color5:#8579C5
-color6:#7A84DF
-color7:#d8cadf
-color8:#19229c
-color9:#b7b1ff
-color10:#a2a6ff
-color11:#c0c4ff
-color12:#657fff
-color13:#9e88ff
-color14:#8899ff
-color15:#fff2ff
-text_color:#fff2ff
-selected_bg_color:#7e6ccb
-selected_fg_color:#fff2ff
-tooltip_bg_color:#131971
-tooltip_fg_color:#fff2ff
-titlebar_bg_color:#131971
-titlebar_fg_color:#fff2ff
-menu_bg_color:#131971
-menu_fg_color:#fff2ff
-link_color:#7e6ccb"
+## Base16 {{scheme-name}}
+# Author: {{scheme-author}}
+# In file gtk-2.0/gtkrc, delete "gtk-color-scheme" including everything between quotes (don't delete gtk-auto-mnemonics) and inject this.
+gtk-color-scheme = "bg_color:#{{base00-hex}}
+color0:#{{base00-hex}}
+text_color:#{{base05-hex}}
+selected_bg_color:#{{base02-hex}}
+selected_fg_color:#{{base05-hex}}
+tooltip_bg_color:#{{base00-hex}}
+tooltip_fg_color:#{{base05-hex}}
+titlebar_bg_color:#{{base00-hex}}
+titlebar_fg_color:#{{base05-hex}}
+menu_bg_color:#{{base00-hex}}
+menu_fg_color:#{{base05-hex}}
+link_color:#{{base02-hex}}"
 gtk-auto-mnemonics	= 1
 
 # Default Style
